website/blog/four-approaches-for-documenting-apis.md

```diff
@@ -0,0 +1,126 @@
+
+### Pros 
+
+**More accurate:** Because your API is generated from existing code, the definition will always remain consistent with the API implementation. Consumers can be confident that the documentation is an accurate reflection of the code and that the API will work as expected. 
+
+**Faster delivery:** If your projects are small or need to be fast to market, the Code First approach might be a better option for you. Developers can get to work immediately, coding directly from requirements documents instead of after the design process, leaving documentation generation until the end of the project.
+
+### Cons 
+
+**Changes are hard to see:** Since changes occur in the code first and are then reflected in the generated API documents, there’s no way to compare previous versions of the API with the current version to validate those changes. 
+
+**Must call each endpoint to generate docs:** To generate docs, you still need to fill in the URI; method; parameters, if any; headers; and authentication details for the API. 
+
+## Method 4: Documentation Diffs with Optic
+
+[Optic](https://useoptic.com/) is a version control system and central repository for API contracts that function much like GitHub. It helps you document your APIs as you build them by observing development traffic to learn the behavior of your API. It presents the API changes it finds as diffs that can be reviewed, approved, or documented before your API is released. 
+
+### How It Works
+
+Using Optic, you can publish your API documentation to a shared website, making it accessible by your entire team. The team can then search documentation, ask questions, see changelogs, and receive change notifications for specific APIs. Optic can manage your documentation process regardless of whether you are using a local or cloud environment. 
+
+![Optic supports design-first and code-first workflows](https://i.imgur.com/AIV6HXO.png)
+
+### Pros
+
+**Accurate:** Because Optic observes changes to your API in real time, and learns from your API contract’s behavior, it’s always in sync with your code. Once you approve changes, Optic updates the specification for the affected APIs. 
+
+**Shows you diffs:** Optic diffs give you the opportunity to compare differences in the local development environment to the API specification. Being able to see differences side by side means that you can isolate potential problems early, ensuring that your API remains robust. 
+
+**Can be used during code review:** Optic provides both API changes and changelogs that can provide valuable information and feedback when reviewing code. 
+
+**Works with all major languages and frameworks:** Integrates with major languages and API frameworks, including Django, Express.js, C-Sharp, and Laravel, IDEs such as IntelliJ, and CI/CD environments like GitHub Actions and CircleCI. 
```

Comment:
I think this list can include more SEO, for example vs code, or at least link to our docs for what we support

Reply:
Added VSCode, link to our frameworks in upcoming commit